tailoff/css/site/base/base.css

```diff
@@ -126,9 +126,9 @@
   transition: opacity 300ms;
 } */
 
-input[type="search"]::-webkit-search-decoration,
```

Comment:
Hebben die dubbele/enkele quotes een impact hier @arnaud-dbu? of hebben die niks met deze change te maken?